src/STEPToMesh.cpp

```diff
@@ -128,6 +130,11 @@ void write(const std::string& outFile, const std::vector<NamedSolid>& namedSolid
 	if (!writer.Write(compound, outFile.c_str())) throw std::logic_error{std::string{"Could not write '"} + outFile + "'"};
 }
 
+std::string str_toupper(std::string s) {
+	std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c) { return std::toupper(c); } );

```

Comment:
This needs to be tested on a localized system (e.g. Turkish) because it is locale dependent.

Or should we remove this altogether and require upper case unit names on the command line (like `-u INCH`) instead?

Reply:
This works also with the Windows locale set to Turkish, because the default locale is the `C` locale.